astroquery/cosmosim/core.py

```diff
@@ -379,8 +387,8 @@ def check_all_jobs(self, phase=None, regex=None, sortby=None):
             if not phase and not regex:
                 if not sortby:
                     t = Table()
-                    t['JobID'] = self.job_dict.keys()
-                    t['Phase'] = self.job_dict.values()
+                    t['JobID'] = [i for i in self.job_dict.keys()]
```

Comment:
Ah, yes. I had this in another branch but forgot to add it here. Oddly enough, casting to a list fails. I had to actually do a list comprehension to fix this.